src/framework/ui/datepicker/datepicker.component.tsx

```diff
@@ -55,7 +58,15 @@ export class DatepickerComponent<D> extends BaseDatepickerComponent<D, CalendarP
 
   static styledComponentName: string = 'Datepicker';
 
-  public getComponentTitle(): string {
+  static defaultProps = {
+    dateService: new NativeDateService(),
+  };
+
+  protected formatDateToString(date: D): string {
+    return this.props.dateService.format(date, FULL_DATE_FORMAT_STRING);
```

Comment:
Since we use the same format in both pickers, let's also move this method to base class too, as soon as `dateService` is moved